iothub/device/src/Transport/AmqpIoT/AmqpIoTConstants.cs

```diff
@@ -39,6 +40,13 @@ internal static class AmqpIoTConstants
         internal const string MethodName = "IoThub-methodname";
         internal const string Status = "IoThub-status";
 
+        public static readonly AmqpSymbol TrackingId = Vendor + ":tracking-id";
+        public static readonly AmqpSymbol ClientVersion = Vendor + ":client-version";
+        public static readonly AmqpSymbol ApiVersion = Vendor + ":api-version";
+        public static readonly AmqpSymbol ChannelCorrelationId = Vendor + ":channel-correlation-id";
+        public static readonly AmqpSymbol AuthChain = Vendor + ":auth-chain";
```

Comment:
What are line 46 and 47 for? Actually can you explain each one of these with a comment?

Reply:
sure, I'll add some comments for each of these.